lib/Doctrine/DBAL/Driver/IBMDB2/DB2Connection.php

```diff
@@ -175,4 +175,18 @@ public function errorInfo()
             1 => $this->errorCode(),
         );
     }
+
+    /**
+     *
+     * Retrieves ibm_db2 native resource handle.
+     *
+     * Could be used if part of your application is not using DBAL.
+     *
+     * @return resource
+     */
+    public function getConn() //getWrappedResourceHandle
```

Comment:
I followed the same behaviour as MysqliConnection->getWrappedResourceHandle()
On IBMi systems is necessary to have access to the resource object to use the native XMLToolkit which is a php library used to execute IBMi programs, change system libraries, etc.